dev-util/bitrise/bitrise-1.35.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit golang-build
+
+EGO_ON="github.com/bitrise-io"
+EGO_PN="${EGO_ON}/${PN}"
+
+DESCRIPTION="Run your Bitrise.io automations on any Mac or Linux machine"
+HOMEPAGE="https://www.bitrise.io/cli"
+SRC_URI="https://${EGO_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT Apache-2.0 BSD ISC MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc"
+RESTRICT="test"
+
+RDEPEND=">=dev-util/envman-2.2.8
+	>=dev-util/stepman-0.11.10"
+
+S="${WORKDIR}/src/${EGO_ON}/${PN}"
+
+src_unpack() {
+	default
+	mkdir -p "${WORKDIR}/src/${EGO_ON}" || die "Couldn't create project dir in GOPATH"
+	mv "${WORKDIR}/${P}" "${WORKDIR}/src/${EGO_ON}/bitrise" || die "Couldn't move sources to GOPATH"
+}
+
+src_compile() {
+	GOPATH="${WORKDIR}" go build -v -o bin/bitrise || die "Couldn't compile bitrise"
+}
+
+src_install() {
+	dobin bin/bitrise
+	dodoc README.md
+	use doc && dodoc -r _docs
```

Comment:
Since these don't get built and don't pull any external dependencies maybe you can just always intall the docs? What's the directory size?

Reply:
OK, will remove condition. I haven't measured the size but it shouldn't be so large: https://github.com/bitrise-io/bitrise/tree/master/_docs